srcpkgs/python3-shapely/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-shapely'
+pkgname=python3-shapely
+reverts="1.7a1_1 1.7a1_2 1.7a1_3"
+version=1.7.0
+revision=1
+wrksrc="Shapely-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Cython"
+makedepends="python3-devel python3-numpy geos-devel"
+checkdepends="python3-pytest"
+short_desc="Library for manipulation and analysis of geometric objects for Python3"
+maintainer="Karl Nilsson <karl.robert.nilsson@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/Toblerity/Shapely"
+distfiles="https://github.com/Toblerity/Shapely/archive/${version}.tar.gz"
+checksum=0f4283d5591578340e36f649736b496d25a793a45b9a2c93b128d2a856f2089b
+nocross="Build is crazy"
```

Comment:
lol

Reply:
The cross-build fails to find the geos library.
I couldn't figure it out, but you're welcome to give it a shot.

Reply:
nah ;)